README.md

```diff
@@ -35,40 +40,295 @@ These playbooks install and configure Wazuh agent, manager and Elastic Stack.
+      become: yes
+      become_user: root
+      vars:
+        wazuh_manager_config:
+          connection:
+              - type: 'secure'
+                port: '1514'
+                protocol: 'tcp'
+                queue_size: 131072
+          api:
+              https: 'yes'
+          cluster:
+              disable: 'no'
+              node_name: 'worker_01'
+              node_type: 'worker'
+              key: 'c98b62a9b6169ac5f67dae55ae4a9088'
+              nodes:
+                  - '"{{ hostvars.manager.private_ip }}"'
+              hidden: 'no'
+        filebeat_output_elasticsearch_hosts:
+                - "{{ hostvars.es1.private_ip }}"
+                - "{{ hostvars.es2.private_ip }}"
+                - "{{ hostvars.es3.private_ip }}"
+  
+  #ODFE+Kibana node
+    - hosts: kibana
+      roles:
+        - role: "../roles/opendistro/opendistro-elasticsearch"
+        - role: "../roles/opendistro/opendistro-kibana"
+      become: yes
+      become_user: root
+      vars:
+        elasticsearch_network_host: "{{ hostvars.kibana.private_ip }}"
+        elasticsearch_node_name: node-6
+        elasticsearch_node_master: false
+        elasticsearch_node_ingest: false
+        elasticsearch_node_data: false
+        elasticsearch_cluster_nodes:
+            - "{{ hostvars.es1.private_ip }}"
+            - "{{ hostvars.es2.private_ip }}"
+            - "{{ hostvars.es3.private_ip }}"
+        elasticsearch_discovery_nodes:
+            - "{{ hostvars.es1.private_ip }}"
+            - "{{ hostvars.es2.private_ip }}"
+            - "{{ hostvars.es3.private_ip }}"
+        kibana_node_name: node-6
+        wazuh_api_credentials:
+          - id: default
+            url: https://{{ hostvars.manager.private_ip }}
+            port: 55000
+            user: foo
+            password: bar
+        instances:
+          node1:
+            name: node-1       # Important: must be equal to elasticsearch_node_name.
+            ip: "{{ hostvars.es1.private_ip }}"   # When unzipping, the node will search for its node name folder to get the cert.
+          node2:
+            name: node-2
+            ip: "{{ hostvars.es2.private_ip }}"
+          node3:
+            name: node-3
+            ip: "{{ hostvars.es3.private_ip }}"
+          node4:
+            name: node-4
+            ip: "{{ hostvars.manager.private_ip }}"
+          node5:
+            name: node-5
+            ip: "{{ hostvars.worker.private_ip }}"
+          node6:
+            name: node-6
+            ip: "{{ hostvars.kibana.private_ip }}"
+```
 
-1. Get the `wazuh-ansible` folder from the `wazuh-qa` [repository](https://github.com/wazuh/wazuh-qa/tree/master/ansible/wazuh-ansible).
+### Inventory file
 
-```
-git clone https://github.com/wazuh/wazuh-qa
+The inventory file sets the public and private address of each node. The public addresses are used to gather facts and provision the instances while the private addresses are used for the cluster communications.
```

Comment:
I'd change the `public IP` and `private IP` statements because they can also be a reference to the FQDN. Also, they can be the same in private environments.

Reply:
Fixed on 8990e2864d1e070dea74ef4e899ccac1e88d67dc